src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -603,10 +604,15 @@ public boolean allFieldsPresent(Collection<OrFields> fields, BibDatabase databas
     /**
      * Returns a clone of this entry. Useful for copying.
      * This will set a new ID for the cloned entry to be able to distinguish both copies.
+     * Ensures that the changed flag is set when the entry type equals the default entry type Misc
      */
     @Override
     public Object clone() {
         BibEntry clone = new BibEntry(type.getValue());
+        if (StandardEntryType.Misc.equals(type.getValue())) {
+            clone.changed = true;
+        }
+
         clone.fields = FXCollections.observableMap(new ConcurrentHashMap<>(fields));
         return clone;
```

Comment:
Isn't it possible to clone the serializatio, too. Then, the issue should be fixed?

Reply:
The issue is already fixed with the changed = true. But I added it as well now